### PR TITLE
Switch to Apache license

### PR DESCRIPTION
**Pull request by muselesscreator**
**Description:** 
Update to apache license based on https://open.edx.org/trademark-licensing-details/#licensing

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[1.0.2] - 2020-09-14
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Move to Apache License
+
 [1.0.1] - 2020-09-14
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Minor updates to attrs, code-annotations, and freezegun dependencies
```
Comment (matthugs) [inline CHANGELOG.rst:23]: I suspect we should move these

**Review by matthugs — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -60,7 +60,7 @@ and are on a command prompt inside the LMS container.
 License
 -------
 
-The code in this repository is licensed under the AGPL 3.0 unless
+The code in this repository is licensed under the Apache unless
```
Comment (nedbat) [inline README.rst:63]: "Apache v2 License"

File: setup.py [hunk 3]
```diff
@@ -75,15 +75,15 @@ def is_requirement(line):
     ],
     include_package_data=True,
     install_requires=load_requirements('requirements/base.in'),
-    license="AGPL 3.0",
+    license="Apache",
```
Comment (nedbat) [inline setup.py:78]: `"Apache 2.0"` I think?

File: README.rst (hunk 2 above)
Comment (muselesscreator) [reply README.rst:63]: fair.

File: CHANGELOG.rst (hunk 1 above)
Comment (muselesscreator) [reply CHANGELOG.rst:23]: not sure what you mean...

File: setup.py (hunk 3 above)
Comment (muselesscreator) [reply setup.py:78]: no idea.  is not included in the codejail setup.py.  Happy to update to this if that is your preference.

File: CHANGELOG.rst (hunk 1 above)
Comment (matthugs) [reply CHANGELOG.rst:23]: I mistook the section levels & thought these were under "Unreleased". Empty sections mess with me.

File: README.rst [hunk 4]
```diff
@@ -60,7 +60,7 @@ and are on a command prompt inside the LMS container.
 License
 -------
 
-The code in this repository is licensed under the Apache unless
+The code in this repository is licensed under the Apache v2 unless
```
Comment (nedbat) [inline README.rst:63]: +"License", so that's it's an English sentence :)  (sorry to be a stickler).  Then it's good to go.

**Pull request merged**